src/main/java/frc/robot/subsystems/Camera/Camera.java

```diff
@@ -27,14 +27,16 @@ public void simulationPeriodic() {
     // This method will be called once per scheduler run during simulation
   }
 
+  //does this work?
   public PhotonPipelineResult getCameraResult() {
-    PhotonPipelineResult result = camera.getLatestResult();
+     PhotonPipelineResult result = camera.getLatestResult();
     return result;
   }
 
+ 
   public PhotonTrackedTarget getCameraTarget() {
     PhotonPipelineResult result = this.getCameraResult();
-    PhotonTrackedTarget target = result.getBestTarget();
```

Comment:
So you have a method for getting the camera result, can you use that instead of using result again?

Reply:
Is this what you meant?
` public double getAngle() {
    PhotonTrackedTarget target = getCameraTarget(); 

    if(target == null) {
        return 0.0;
    }
    else {  
    double angle = Math.toRadians(target.getYaw());
    return angle;
    }
  }`

Reply:
this is fine